File: JavaScript/2.Strings-Basics/1.Reverse String.js
//Reverse String
//example text="hello"; output ="olleh"


const Reverse = (text) => {

    if (text === ""){
        return
    }

    let ReverseText = ""

    for(let i = text.length - 1; i >= 0; i--){

        ReverseText += text[i]
    }

    return ReverseText
}

const text = "hello"

console.log("Reversed String :", Reverse(text))

File: Python/2.String-Basics/1.Reverse String.py
# Reverse a String
# Examples
# reverse_string("Hello World") ➞ "dlroW olleH"

def reverse_string(text):

    reversed_text = ""

    for i in range(len(text)-1, -1, -1):
        reversed_text += text[i]

    return reversed_text


print(reverse_string("Hello World"))


